single_tech_samples/synapseanalytics/sample3_integration_testing/README.md

```diff
@@ -0,0 +1,180 @@
+# DataOps - Synapse Integration Testing <!-- omit in toc -->
+
+This sample showcases additional integration tests for Azure Synapse. It includes the deployment of infrastructure based on the E2E sample [DataOps - Parking Sensor (Synapse)](https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/main/e2e_samples/parking_sensors_synapse).
+
+## Contents <!-- omit in toc -->
+
+- [Solution Overview](#solution-overview)
+  - [Technologies used](#technologies-used)
+- [Key Concepts](#key-concepts)
+  - [Environments](#environments)
+  - [Testing](#testing)
+- [How to use the sample](#how-to-use-the-sample)
+  - [Prerequisites](#prerequisites)
+    - [Software pre-requisites if you use dev container](#software-pre-requisites-if-you-use-dev-container)
+  - [Setup and Deployment](#setup-and-deployment)
+    - [Deployed Resources](#deployed-resources)
+    - [Clean up](#clean-up)
```

Comment:
I feel that there should be a section that directs the reader to the integration test README prior to letting them know about the clean up script.

That way the reader knows that after a successful deployment, they can open up the integration test codebase (via devcontainer or not), and run the integration test bed

Reply:
updated!